### ISSUE TITLE
flutter/flutter main and master branches no longer pointing to the same commit

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#154136]
These branches should be in-sync.
At the moment, master is pointed to 114dd89 (up-to-date) and main is pointed to b6c14d7 (4 days ago).

The mirror GitHub action is failing with Bad credentials
https://github.com/flutter/flutter/actions/runs/10566254966/job/29272595079#step:2:23

```
Run google/mirror-branch-action@30c52ee21f5d3bd7fb28b95501c11aae7f17eebb
  with:
    github-token: ***
    source: master
    dest: main
Error: Bad credentials
```
The same thing happened 2 years ago:
https://github.com/flutter/flutter/issues/105159

> Could you help renew the token `FLUTTERMIRRORINGBOT_TOKEN`?

> Updated the token and ran the failing workflows. Now main and master point to the same commit.

https://github.com/flutter/flutter/blob/114dd89bbb831d7a48ba15287dcb34b6c0147762/.github/workflows/mirror.yml#L24-L26

**Comment by jmagman** [comment]
Paper trail for the mirroring bot go/flutter-github-actions#authentication

**Comment by christopherfujino** [comment]
Ok, I generated a new token and updated the `FLUTTERMIRRORINGBOT_TOKEN` secrets key in https://github.com/organizations/flutter/settings/secrets/actions.

I found the easiest way to generate a new token was to log into the https://github.com/fluttermirroringbot account -> settings -> developer settings -> Personal access tokens -> Tokens (classic) -> select the pre-existing, expired token -> "Regenerate token" button.

By using the "regenerate token" button, it will ensure the new token has all of the same permissions as the old one.
